src/AirshipBindings.NETStandard/AirshipBindings.NETStandard.Android/Airship.cs

```diff
@@ -211,12 +213,26 @@ public void AssociateIdentifier(string key, string identifier)
 
         public void DisplayMessageCenter()
         {
-            MessageCenterClass.Shared().ShowMessageCenter();
+            if (OnMessageCenterDisplay != null)
+            {
+                OnMessageCenterDisplay.Invoke(this, new MessageCenterEventArgs());
+            }
+            else
+            {
+                MessageCenterClass.Shared().ShowMessageCenter();
```

Comment:
You should only have to call this here. If you set the message center listener on the airship sdk, it will forward to the right place if set - https://docs.airship.com/platform/android/message-center/#overriding-the-default-display

Reply:
The approach should be, if you have a netstandard listener, set the SDK listener, otherwise clear it.